### PR TITLE
Polish the content and styling of the site

### PR DESCRIPTION
**Pull request by haoyxc**
Make a series of small changes to give the site more context as well and also make it seem neater.

A few things I did were: change the font, decreased spacing between certain areas of text, changed the skills, and remove unused code. 

A peak at the content:
image
image
cindyhao-step-2020.appspot.com

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,12 +2,12 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio.</title>
+    <title>Cindy Hao</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```
Comment (fenghaolw) [inline portfolio/src/main/webapp/index.html:7]: Should we load jquery before script.js? It seems like script.js depends on jquery.

You might find this interesting to read:
https://www.html5rocks.com/en/tutorials/speed/script-loading/

The concept of script loading and execution order is critical for frontend performance (and our team does a lot of work in this area :))

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -52,7 +84,7 @@
 
 /* Style the header links */
 .header a {
-  float: left;
+  /* float: left; */
```
Comment (fenghaolw) [inline portfolio/src/main/webapp/style.css]: ?

Comment (haoyxc) [reply portfolio/src/main/webapp/style.css]: OH OOPS

Comment (haoyxc) [reply portfolio/src/main/webapp/style.css]: I think I was just playing around to see if the float made any impact and forgot to delete it -- fixed!

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (haoyxc) [reply portfolio/src/main/webapp/index.html:7]: OH wow ok I didn't know that! thanks for letting me know!

**Review by fenghaolw — APPROVED** [review]

**Pull request merged**